src/System.Drawing.Common/tests/mono/System.Drawing/GraphicsTests.cs

```diff
@@ -379,44 +389,46 @@ private void Compare(string msg, RectangleF b1, RectangleF b2)
         [ConditionalFact(Helpers.GdiplusIsAvailable)]
         public void Clip_GetBounds()
         {
-            Graphics g = Get(16, 16);
-            RectangleF bounds = g.Clip.GetBounds(g);
-            Assert.Equal(0, bounds.X);
-            Assert.Equal(0, bounds.Y);
-            Assert.Equal(16, bounds.Width);
-            Assert.Equal(16, bounds.Height);
-            Assert.True(g.Transform.IsIdentity);
-            g.Dispose();
+            using (Graphics g = Get(16, 16))
```

Comment:
This will leak the `Bitmap` object that the `Graphics` is attached to -- `Dispose`ing the Graphics will not affect the `Bitmap`. There's a couple options:

* Have `Get()` return some kind of compound Disposable object which disposes both its `Graphics` and `Bitmap`
```
class BitmapAndGraphics : IDisposable
{
    private readonly Bitmap _bitmap;
    public Graphics Graphics { get; }
    public BitmapAndGraphics(int width, int height) {
        _bitmap = new Bitmap(width, height); Graphics = Graphics.FromImage(_bitmap); Graphics.Clip = new Region(new Rectangle(0, 0, width, height));
    }
    public void Dispose() { Graphics.Dispose(); _bitmap.Dispose(); }
}
```

* Manually create the Bitmap and Image in each of these tests.